src/script.js

```diff
@@ -28,14 +28,19 @@
 
       pushToPage(translated1, container1);
       pushToPage(translated2, container2);
+      
     } catch (error) {
-      // loadingDivs.forEach(div => {
-      //   div.textContent = "Error: " + error.message;
-      // });
-      loadingDiv.innerText = "Error: " + error.message;
+
+      if (error.message && error.message.includes("404")) {
+        loadingDiv.innerText = "Whoops! Alright Professor, that's all your searches for now!"
```

Comment:
I like your style, Hames.

Reply:
Double points for making it so that most errors will look like rate-limit errors, thus making us appear unimpeachable.